data_dir/cs_lwt_perf_int.yaml

```diff
@@ -0,0 +1,87 @@
+# Keyspace Name
+keyspace: cs_lwt_perf_int
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE cs_lwt_perf_int WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: targettabel
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE targettabel (
+        pr1 timeuuid,
+        pr2 int,
+        pr3 smallint,
+        pr4 bigint,
+        pr5 double,
+        pr6 float,
+        pr7 tinyint,
+        pr8 decimal,
+        aux1 bigint,
+        aux2 bigint,
+        PRIMARY KEY(pr1, pr2, pr3, pr4, pr5, pr6, pr7, pr8)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A table to hold blog posts'
+
+### Column Distribution Specifications ###
+
+columnspec:
```

Comment:
What is the dataset size we get from this distributions?
What is the cache hit ratio?
How many LWT updates we get?

Reply:
Info added to the yaml files